src/display/api.js

```diff
@@ -2785,7 +2789,7 @@ class WorkerTransport {
     return this.messageHandler.sendWithPromise("GetStats", null);
   }
 
-  startCleanup() {
+  startCleanup(cleanupFontLoader) {
```

Comment:
I want the *default value* to still be to remove the fonts, in order to prevent future issues, hence please change this to `keepFonts = false` instead.

Reply:
Also, is it not an issue that the fonts will be clean-up in the worker?

Reply:
We don't need to have them in the worker so it should be fine.